src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -147,6 +148,44 @@ public void setPerson(Student target, Student editedStudent) {
         public void updateFilteredPersonList(Predicate<Student> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public ObservableList<Session> getFilteredSessionList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
```

Comment:
Actually these methods should be in a separate test class called AddSessionCommandTest. You can refer to the format of the original AddCommandTest!